frontend/src/Component/ApplicationForm.jsx

```
@@ -0,0 +1,111 @@
+                        <label className="mb-1 capitalize text-gray-300">{ph.replace(/_/g, ' ')}</label>
+                        <input
+                            type="text"
+                            name={ph}
+                            value={values[ph] || ''}
+                            onChange={handleChange}
+                            required
+                            className="bg-gray-800 text-white border border-gray-600 rounded px-3 py-2 focus:outline-none focus:ring-2 focus:ring-blue-500"
+                        />
+                    </div>))}
+                <div className="md:col-span-2 flex flex-col">
+                    <label className="mb-1 text-gray-300">Attachment (optional):</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)